pip_requirements.txt

```diff
@@ -7,7 +7,7 @@ setuptools-scm==5.0.2
 setuptools-rust==0.11.4
 cryptography==3.2
 jinja2==2.10.1
-ansible==2.9.2
+ansible==2.9.7
```

Comment:
We should update documentation saying minimum ansible version 2.9.7. Also upgrade setup script to install 2.9.7. Test with container

Reply:
Where should I document this?

Reply:
In src/playbooks/build we should make sure minimum ansible version is 2.9.7 
We should also add couple of lines in release notes, one for this feature and one for ansible version change. 

Also, document in Readme.md after this line
MetroAE is based off of the Python-based Ansible operations tool.

Saying minimum ansible supported version is 2.9.7

Reply:
Since 2.9.7 is only required to deploy NUH on vcenter, do we need to enforce it through build (especially since build checks to see if the ansible version is >= 2.9.2) or would documentation updates be sufficient?

Reply:
Okay, then we should assert in the build vars after Assert that NUH version is at least 20.10.R5, so that it fails in the build only specific to the deployment if NUH on vcenter is defined and ansible less than 2.9.7 instead of failing in deploy/predeploy.

Reply:
I have the NUH version check in nuh-process-vars at the moment. Should I add an Ansible version check there too?

Reply:
Yes please!

Reply:
Done! I added a release note as well.